apple/repositories.bzl

```diff
@@ -109,11 +109,10 @@ def apple_rules_dependencies(ignore_version_differences = False):
     _maybe(
         http_archive,
         name = "build_bazel_apple_support",
-        sha256 = "36d60bce680446ab534b141c47f2aef6b9c598267ef3450b7d74b9d81e1fd6bd",
-        urls = [
-            "https://github.com/bazelbuild/apple_support/releases/download/0.9.0/apple_support.0.9.0.tar.gz",
-        ],
         ignore_version_differences = ignore_version_differences,
+        sha256 = "c5f232e8fe0cfc375eecf93f2d31f45fb7cf6a5060270563b1ed5ffad6f65c26",
+        strip_prefix = "apple_support-b7047faee98aae74b76618d618beadc28cbdb10e",
+        url = "https://github.com/bazelbuild/apple_support/archive/b7047faee98aae74b76618d618beadc28cbdb10e.tar.gz",
```

Comment:
This will be reverted before landing, but currently it includes https://github.com/bazelbuild/apple_support/pull/67